Disable "Edit on GitHub" button for all changelog pages

lluiscampos opened the pull request:
The content is auto-generated. Furthermore the links would point to the
wrong place (mender-docs repository).

tranchitella reviewed (changes requested):
To make the template easier to write, I changed the syntax to `github: false`.
See here: https://github.com/mendersoftware/mender-docs-site/blob/master/docbook_to_markdown.py#L120

lluiscampos commented:
> To make the template easier to write, I changed the syntax to `github: false`.
> See here: https://github.com/mendersoftware/mender-docs-site/blob/master/docbook_to_markdown.py#L120

Right! I knew we changed it but because I took mender-docs as the example, I failed to see which was the final one. We need to fix (actually, just remove?) [these](https://github.com/mendersoftware/mender-docs/pull/1461/files#diff-59152cd36a57eac52946f23029daef3a64a05e24a7c23b9c03955cc078eba788R5-R6).

tranchitella commented:
> > To make the template easier to write, I changed the syntax to `github: false`.
> > See here: https://github.com/mendersoftware/mender-docs-site/blob/master/docbook_to_markdown.py#L120
> 
> Right! I knew we changed it but because I took mender-docs as the example, I failed to see which was the final one. We need to fix (actually, just remove?) [these](https://github.com/mendersoftware/mender-docs/pull/1461/files#diff-59152cd36a57eac52946f23029daef3a64a05e24a7c23b9c03955cc078eba788R5-R6).

Yes, those are left-overs.
